webinterface/_base.py

```diff
@@ -35,15 +35,24 @@ def _preface(self):
             """,
             unsafe_allow_html=True,
         )
+        st.image("https://github.com/Proteobench/ProteoBench/blob/Add-logos/webinterface/logos/logo_participants/logos_all_20230926.png?raw=true")
```

Comment:
I guess you discussed this, but what is the reason to use the github url? Wouldn't the application be shipped with the images and the relative path be enough?

Reply:
We did not discuss this. I just did it this way for lack of knowing better I think. 
I am totally open to suggestions :)